### DISCUSSION TITLE
what hapenned to the custom folder?

### DISCUSSION THREAD

**Discussion by Airlegendz** [NvChad/NvChad#2925] [Q&A] [answered]
Hello I was following a tutorial on adding some plugins for autosuggestion for c++ but I realized that there was not a folder titled custom in ~/.config/nvim/lua there still is chadrc.lua 
so my question is how do i install plugins in the current version

**Comment by siduck** [comment, marked as answer]
@Airlegendz custo syntax is no longer needed. now user can just use any config and use nvchad's main repo itself as a plugin and import its modules, check the news page for v2.5.

and you dont need to follow tutorials! just enable clangd lsp and completion will work
